pkg/policy/execute.go

```diff
@@ -245,15 +245,24 @@ func (e *Executor) Execute(ctx context.Context, req *ExecuteRequest, policy *Pol
 	return &total, nil
 }
 
-func (e *Executor) createCheckResult(ctx context.Context, selector []string, policyExecution *state.PolicyExecution, q *Check, qr *QueryResult, err error) error {
+func normalizeCheckSelector(policyExecution *state.PolicyExecution, policyPath []string, policyName string) string {
+	selector := []string{policyExecution.PolicyName}
+	if !strings.HasPrefix(selector[0], policyExecution.Location+"//") {
+		selector[0] = policyExecution.Location + "/"
```

Comment:
Not sure I get the logic of overwriting policyName here, and then appending it in L254 regardless of if it was overridden or not.

Reply:
Good catch. I made a mistake in naming... `policyName` should be `checkName`